### DISCUSSION TITLE
Search with "priority" parameters

### DISCUSSION THREAD

**Discussion by skiiiks** [qdrant/qdrant#3069] [Q&A]
Is there any way to sort (not to filter, but sort) results in a search BASED on a payload metadata ?

I have all my embeddings with a payload called "priority", and I would like the search method to prioritize first the ones with higher "priority" 😉

**Comment by timvisee** [comment]
> Is there any way to sort (not to filter, but sort) results in a search BASED on a payload metadata ?

No, this is currently not possible.

In your case, would it be possible to fetch all and sort in your client after fetching all results?
